erizo/src/erizo/rtp/RtpSlideShowHandler.cpp

```diff
@@ -70,22 +69,16 @@ void RtpVP8SlideShowHandler::write(Context *ctx, std::shared_ptr<dataPacket> pac
     slideshow_seq_num_ = last_original_seq_num_;
   }
   if (slideshow_is_active_) {
-    RtpVP8Parser parser;
-    unsigned char* start_buffer = reinterpret_cast<unsigned char*> (packet->data);
-    start_buffer = start_buffer + rtp_header->getHeaderLength();
-    RTPPayloadVP8* payload = parser.parseVP8(
-        start_buffer, packet->length - rtp_header->getHeaderLength());
-    if (!payload->frameType) {  // Its a keyframe
-      sending_keyframe_ = true;
+    bool is_keyframe = false;
+    RtpMap *codec = connection_->getRemoteSdpInfo().getCodecByExternalPayloadType(rtp_header->getPayloadType());
```

Comment:
I'm thinking maybe we could store the corresponding external PTs in notifyUpdate so we don't have to look for the PT for every packet.